services/blockchain-connector/shared/utils/regex.js

```diff
@@ -17,10 +17,20 @@ const BLS_KEY = /^\b[a-fA-F0-9]{96}\b$/;
 const CHAIN_ID = /^\b[a-fA-F0-9]{8}\b$/;
 const PROOF_OF_POSSESSION = /^\b[a-fA-F0-9]{192}\b$/;
 const KEY_NOT_EXIST = /^Key [a-f0-9]+ does not exist\.$/;
+const MODULE = /^\b(?:[\w!@$&.]{1,32}|,)+\b$/;
+const COMMAND = /^\b(?:[\w!@$&.]{1,32}|,)+\b$/;
+const ADDRESS_LISK32 = /^lsk[a-hjkm-z2-9]{38}$/;
+const NONCE = /^[0-9]+$/;
```

Comment:
Not needed, just sharing an alternative.

```suggestion
const NONCE = /^\d+$/;
```